package.json

```diff
@@ -23,7 +23,15 @@
     "merge:lcov": "cat ./coverage/*/lcov.info > ./coverage/lcov.info",
     "coverage": "npm run merge:lcov && cat ./coverage/lcov.info | coveralls",
     "heroku-postbuild": "npm run build",
-    "watch:scss": "gulp watchSCSS"
+    "watch:scss": "gulp watchSCSS",
```

Comment:
Do we still use this task?  I thought we have moved watching scss file changes to the `gulp startDev` task.

Reply:
Yes we have. I forgot to remove it. I'll remove that immediately. Thanks.